Kconfig

```diff
@@ -1489,6 +1489,20 @@ menu "LVGL configuration"
 			bool "Use evdev input driver"
 			default n
 
+		config LV_USE_LIBINPUT
+    		bool "Use libinput input driver"
+			default n
+
+		config LV_LIBINPUT_BSD
+			bool "Use the BSD variant of the libinput input driver"
+			depends on LV_USE_LIBINPUT
+			default n
+
+		config LV_LIBINPUT_XKB
+			bool "Enable full keyboard support via XKB"
+			depends on LV_USE_LIBINPUT
+			default n
+
```

Comment:
I'm unsure if and how to add `LV_LIBINPUT_XKB_KEY_MAP`. Similar macros like `LV_CALENDAR_DEFAULT_DAY_NAMES` are also absent here. Is this intentional?